constants_test.py

```diff
@@ -29,8 +29,8 @@ class ConstantsTests(test_utils.GenericTestBase):
 
     def test_constants_file_is_existing(self):
         """Test if the constants file is existing."""
-        self.assertTrue(os.path.isfile(os.path.join(
-            'assets', 'constants.ts')))
+        self.assertTrue(
+            os.path.isfile(os.path.join('assets', 'constants.ts')))
```

Comment:
Is there any issue with the old code? are was expecting that to pass the test? (I think yes!)

@bansalnitish & @Hudda Can you please check with the author of the issue or with other developers about the expected behaviour for this check?

Reply:
Confirming that the changed version is a bit nicer, but the old behaviour should pass the test.

Reply:
@Hudda please could you confirm whether the previous formatting would have passed the lint check you implemented?

Reply:
No, the tests implemented by Hudda would not allow the old behaviour to pass.

Reply:
Then I think this PR is incorrect and the best option might just be to redo it (as discussed in the meeting).